pkgs/development/tools/nailgun/default.nix

```diff
@@ -8,33 +8,56 @@ let
     inherit version;
     sha256 = "1mk8pv0g2xg9m0gsb96plbh6mc24xrlyrmnqac5mlbl4637l4q95";
   };
-in
-stdenv.mkDerivation {
-  pname = "nailgun";
-  inherit version;
 
-  src = fetchFromGitHub {
-    owner = "facebook";
-    repo = "nailgun";
-    rev = "nailgun-all-v${version}";
-    sha256 = "1syyk4ss5vq1zf0ma00svn56lal53ffpikgqgzngzbwyksnfdlh6";
-  };
+  license = lib.licenses.asl20;
+  homepage = "http://www.martiansoftware.com/nailgun/";
+  platforms = lib.platforms.linux;
+  maintainers = with lib.maintainers; [ ];
 
-  makeFlags = [ "PREFIX=$(out)" ];
+  server = runCommand "nailgun-server-${version}" {
```

Comment:
This should use mkDerivation

Reply:
How would that work? There's no `src` for this; the default builder fails with `variable $src or $srcs should point to the source` when I leave it blank. Should I, like, disable phases until it works or something?

Reply:
You can disabling unpacking with ``dontUnpack = true``.